roles/openshift_health_checker/openshift_checks/package_version.py

```diff
@@ -9,6 +9,9 @@ class PackageVersion(NotContainerizedMixin, OpenShiftCheck):
     name = "package_version"
     tags = ["preflight"]
 
+    recommended_ovs_version_os_3_5 = "2.6"
+    recommended_ovs_version = "2.4"
```

Comment:
hm, I'll go ahead and turn this into a map, but not sure how future versions will be handled. cc @brenton